src/components/RadioButton/radio.module.scss

```diff
@@ -143,6 +143,132 @@
+      color: var(--radio-button-pill-container-active-text-color);
+    }
+
+    label {
+      height: auto;
+      min-height: 36px;
+      padding: $selector-padding-vertical-medium
+        $selector-padding-horizontal-medium;
+    }
+
+    input {
+      & + label {
+        .radio-button {
+          background: var(--radio-button-in-pill-background-color);
+          border: var(--radio-button-in-pill-border);
+
+          &:after {
+            border-color: var(--radio-button-in-pill-pip-color);
+          }
+        }
+      }
+
+      &:not(.disabled):not([disabled]):active + label {
+        .radio-button {
+          background: var(--radio-button-in-pill-active-background-color);
+          border: var(--radio-button-in-pill-active-border);
+        }
+      }
+
+      &:not(.disabled):not([disabled]):hover + label {
+        .radio-button {
+          background: var(--radio-button-in-pill-hover-background-color);
+          border: var(--radio-button-in-pill-hover-border);
+        }
+      }
+    }
+
+    input:checked {
+      & + label {
+        .radio-button {
+          background: var(--radio-button-in-pill-checked-background-color);
+          border: var(--radio-button-in-pill-checked-border);
+        }
+      }
+
+      &:not(.disabled):not([disabled]):active + label {
+        .radio-button {
+          background: var(
+            --radio-button-in-pill-checked-active-background-color
+          );
+          border: var(--radio-button-in-pill-checked-active-border);
+        }
+      }
+
+      &:not(.disabled):not([disabled]):hover + label {
+        .radio-button {
+          background: var(
+            --radio-button-in-pill-checked-hover-background-color
+          );
+          border: var(--radio-button-in-pill-checked-hover-border);
+        }
+      }
+    }
+
+    &:hover:not([disabled]) {
+      background: var(--radio-button-pill-container-hover-background-color);
+      border-color: var(--radio-button-pill-container-hover-border-color);
+      color: var(--radio-button-pill-container-hover-text-color);
+
+      input {
+        &:not(.disabled):not([disabled]) + label {
+          .checkmark {
+            background: var(--radio-button-in-pill-hover-background-color);
+            border: var(--radio-button-in-pill-hover-border);
+          }
+        }
+      }
+    }
+
+    &:active:not([disabled]) {
+      background: var(--radio-button-pill-container-active-background-color);
+      border-color: var(--radio-button-pill-container-active-border-color);
+      color: var(--radio-button-pill-container-active-text-color);
+
+      input {
+        &:not(.disabled):not([disabled]) + label {
+          .radio-button {
+            background: var(--radio-button-in-pill-active-background-color);
+            border: var(--radio-button-in-pill-active-border);
+          }
+        }
+      }
+    }
+
+    &:disabled,
+    &.disabled {
+      opacity: $disabled-alpha-value;
```

Comment:
do we want to make this a css var? Not imagine it would get used much, but might be useful in dark themes

Reply:
Will do a css refactor in a separate change.